doc/release-notes.md

```diff
@@ -1,6 +1,240 @@
 # release notes
 
 ## current release
+- release date: 2023-04-XX
+- status: available
+- overview of changes (See [PR XXX](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/) for additional details):
+  - This release adds the following data: 
+    - 744 methylation array data samples
+      - 728 PBTA samples (17 normal, 711 tumor)
+      - 16 TARGET samples (6 normal, 10 tumor)
+    - 1264 RNA-seq samples
+      - 1264 PBTA samples (3 normal, 1261 tumor)
+    - 2759 WGS/WXS samples
+      - 3 Kentucky samples (3 normal)
+      - 2441 PBTA WGS samples (1171 normal, 1270 tumor)
+      - 315 PBTA WXS samples (160 normal, 155 tumor)
+  
+  - files changed
+    - files added
+      - `20038D-17Q6-01.regions.100bp_padded.bed`
+      - `S0274956_Padded_HG38.merged.bed`
+      - `agilent-v4-targets-ucsc.100bp_padded.bed`
+      - `UCSC_hg19-GRCh37_Ensembl2RefSeq.tsv`
+      - `cnv-gatk.seg.gz`
+      - `fusion-annoFuse.tsv.gz`
+      - `fusion_summary_lgg_hgg_foi.tsv`
+      - `independent-specimens.methyl.primary-plus.eachcohort.tsv`
+      - `independent-specimens.methyl.primary.eachcohort.tsv`
+      - `independent-specimens.methyl.relapse.eachcohort.tsv`
+      - `methyl-md5sum.txt`
```

Comment:
Maybe we can call out the methyl data being provided for separate download due to size